Overview.bs

```diff
@@ -999,11 +1001,18 @@ in an extended font subset:
 
 Additionally:
 
-*  <span class="conform server" id="conform-response-patch-format">The format of the patch in the either the <code>patch</code> or <code>replace</code> fields must be
-    one of those listed in <code>accept_patch_format</code></span>.
+*  <span class="conform server" id="conform-response-patch-format">The format of the patch in the
+    either the <code>patch</code> or <code>replace</code> fields must be one of those listed in
+    <code>accept_patch_format</code></span>.
 
-*  The value of <code>original_font_checksum</code> must be set to the checksum of the original font.
-    The checksum value must computed by the procedure in [[#computing-checksums]].
+*  <span class="conform server" id="conform-response-original-checksum">The value of
+    <code>original_font_checksum</code> must be set to the checksum of the original font.
+    The checksum value must computed by the procedure in [[#computing-checksums]].</psan>
+
+*  <span class="conform server" id="conform-response-patched-checksum">
+     If <code>patch</code> or <code>replacement</code> fields are set the value of
+     <code>patched_checksum</code> must be set to the checksum of the extended font subset.
+     The checksum value must computed by the procedure in [[#computing-checksums]].</span>
 
 *  If the set of codepoints the client has is empty the response must set the
```

Comment:
The statements that are tagged seem good but this patch is for some, not all of them, right?

Reply:
That's right, I've been adding them in parallel with adding the tests for them. So as I continue to add tests the remaining ones will get tagged.